common-library/src/main/java/com/yas/commonlibrary/csv/CsvExporter.java

```diff
@@ -0,0 +1,65 @@
+package com.yas.commonlibrary.csv;
+
+import com.yas.commonlibrary.csv.anotation.CsvColumn;
+import com.yas.commonlibrary.csv.anotation.CsvName;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.lang.reflect.Field;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+
+public class CsvExporter {
+
+    public static <T> byte[] exportToCsv(List<T> dataList, Class<T> clazz) throws IOException {
```

Comment:
I think you should use CSV library (i.e opencsv). `CSVWriter `will help to escapes special characters like ,  \ in the content. In addition, please initialize `Writer`, `ByteArrayOutputStream` in `try` so they will be closed properly.

```
try (ByteArrayOutputStream baos = new ByteArrayOutputStream();
     CSVWriter writer = new CSVWriter(new OutputStreamWriter(baos))) {
```